api_v2/src/leaves/leaves.resolver.spec.ts

```diff
@@ -1,14 +1,28 @@
+import { PrismaService } from '@/prisma/prisma.service';
 import { Test, TestingModule } from '@nestjs/testing';
+import { LeavesRepository } from './leaves.repository';
```

Comment:
```suggestion
import { LeavesRepository } from './leaves.repository';
```
Use proper import pathing